### ISSUE TITLE
The app crashes when adding a new virtual sensor

### ISSUE THREAD

**Issue by berkeragir** [AblionGE/LocPrivLib#17]
When adding a new virtual sensor, I add a stream source (gps), make the settings and then press 'save'. The app crashes at this point. Here is the error stack I get on the Android Studio:

```
11-17 14:54:00.129  21293-21509/tinygsn.gui.android E/SQLiteLog﹕ (1) no such table: vs_location
11-17 14:54:00.139  21293-21509/tinygsn.gui.android E/AndroidRuntime﹕ FATAL EXCEPTION: AsyncTask #8
    Process: tinygsn.gui.android, PID: 21293
    java.lang.RuntimeException: An error occured while executing doInBackground()
            at android.os.AsyncTask$3.done(AsyncTask.java:304)
            at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
            at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
            at java.util.concurrent.FutureTask.run(FutureTask.java:242)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:818)
     Caused by: android.database.sqlite.SQLiteException: no such table: vs_location (code 1): , while compiling: Select * from vs_location test  where timed > 0 order by _id desc limit ?
            at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
            at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:917)
            at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:528)
            at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
            at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
            at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
            at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:68)
            at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1400)
            at android.database.sqlite.SQLiteDatabase.rawQuery(SQLiteDatabase.java:1339)
            at tinygsn.storage.db.SqliteStorageManager.executeQueryGetLatestValues(SqliteStorageManager.java:183)
            at tinygsn.storage.db.SqliteStorageManager.executeQueryGetLatestValues(SqliteStorageManager.java:167)
            at tinygsn.controller.AndroidControllerVS.loadLatestData(AndroidControllerVS.java:73)
            at tinygsn.controller.AndroidControllerVS.loadLatestData(AndroidControllerVS.java:55)
            at tinygsn.gui.android.ActivityListVS$4.doInBackground(ActivityListVS.java:156)
            at tinygsn.gui.android.ActivityListVS$4.doInBackground(ActivityListVS.java:153)
            at android.os.AsyncTask$2.call(AsyncTask.java:292)
            at java.util.concurrent.FutureTask.run(FutureTask.java:237)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:818)
11-17 14:54:00.199  21293-21293/tinygsn.gui.android D/FindExtension﹕ FindExtension: before mHardwareRenderer.initialize, mSurface.isValid() = true
```

**Comment by AblionGE** [comment]
I'll try to reproduce it to understand how I can fix that. Which VS did you select and what options ?

**Comment by berkeragir** [comment]
Interestingly, it worked now. The difference: name of the VS. I was trying with a name that had spaces in it ('location test'). Now, I tried with 'locationTest' and it worked. I tried 'bridge' and 'notification' types btw; the above error occurred in both cases. I tried with several options (including the default). Apparently, it is linked to the name. However, there is no clue that the name should not include spaces, and there is no warning as well.
